src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-user-form/rgw-user-form.component.ts

```diff
@@ -380,6 +381,10 @@ export class RgwUserFormComponent implements OnInit {
     this.userForm.markAsDirty();
   }
 
+  hasCapabilityButtonToBeDisabled() {
+    return this.capabilities.length === RgwUserCapabilities.getAll().length;
```

Comment:
The original code will not catch duplicates in the capability list (which shouldn't happen, but who knows). To be aware of this, simply use the following code:
```suggestion
    return !_.difference(RgwUserCapabilities.getAll(), _.map(this.capabilities, 'type')).length;
```

Reply:
Done.

Reply:
If done, have you forgotten to push the changes?

Reply:
Sorry, Now it's properly updated.